todo_app/templates/index.html

```diff
@@ -2,17 +2,149 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
+  <div class="jumbotron jumbotron-fluid">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">Leon's to-do app 'Today, Ta-Da!'.</p>
+    <input type="button" onclick="alert('v1.1 beta')" value="ABOUT">
   </div>
 
+  <div class="row">
+    <form action="/add-item" method="POST">
+      <div class="form">
+        <input type="submit" value="Add todo item">
+        <br>
+        <label for="title">Todo item title: </label>
+        <input type="text" name="title" id="title" required>
+      </div>
+  </div>  
+  <br><br>
+  </form>
+<!--
+  <div class="row">
+    <form action="/del-item" method="POST">
+      <div class="form">
+        <input type="submit" value="Delete todo item">
+        <label for="title">Todo item id: </label>
+        <input type="number" name="del_id" id="del_id" >
+      </div>
+  </div>  
+  <br><br>
+-->
+
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items_list %}
+          <li>{{item.id}}: {{item.title}} {{item.status}}</li>
+        {% endfor %}
       </ul>
     </div>
+    
+    <form action="/upd-item" method="POST">
+      <br>
+        <div class="form">
+        <input type="submit" value="Update todo item">
+        <br>
+        <label for="title">Todo item id: </label>
+        <input type="number" name="upd_id" id="upd_id" >
+        <br>
+        <label for="title">Updated title: </label>
+        <input type="text" name="title" id="title" >
+        <br>
+        <label for="title">Updated status: </label>
+        <input type="text" name="status" id="status" >
+        <div class="btn-group btn-group-toggle" data-toggle="buttons">
+          <label class="btn btn-secondary active">
+            <input type="radio" name="options" id="option1" autocomplete="off" checked> Not started
+          </label>
+          <label class="btn btn-secondary">
+            <input type="radio" name="options" id="option2" autocomplete="off"> In progress
+          </label>
+          <label class="btn btn-secondary">
+            <input type="radio" name="options" id="option3" autocomplete="off"> Complete
+          </label>
```

Comment:
It looks to me like these are currently being ignored - if you'd rather use them over the free text box (and I'd agree the buttons are nicer!) then we can adjust them slightly by:
* Adding an appropriate `value` to each input: `value="In Progress"` etc.
* Reading that value from the form using `request.form.get("options")` instead of `request.form.get("status")`